HubSpot.NET.Examples/Companies.cs

```diff
@@ -1,16 +1,32 @@
 ﻿using HubSpot.NET.Api.Company;
 using HubSpot.NET.Api.Company.Dto;
 using HubSpot.NET.Core;
+using System;
 
 namespace HubSpot.NET.Examples
 {
     public class Companies
     {
         public static void Example(HubSpotApi api)
+        {
+            try
```

Comment:
Sounds good to me - I'm pretty new to open source, do you have much experience with project planning from an OSS perspective? I wonder if it would be worth setting up a call or a Slack channel etc.?

Reply:
I have a lot of experience project planning and I'm familiar enough with OSS to make project plans, but have I done them both together in a team setting? No, I haven't done that, but I'm very confident that I could. I think having a Slack/Discord/`<enter-chat-client-here>` channel would definitely be beneficial. I tried to set up a Gitter room for `vtr-dev` but Gitter seems to be having some problems.

We would be happy to provide a Slack channel if you would like. What do you think?

Reply:
@clarkd if you'd like to talk more about it, you can email me. I'd be more than happy to talk about it with you.